### ISSUE TITLE
Field continuation is not supported by abcjs

### ISSUE THREAD

**Issue by sjktje** [paulrosen/abcjs#932]
According to the [2.1 ABC standard](https://abcnotation.com/wiki/abc:standard:v2.1#field_continuation), a "field that is too long for one line may be continued by prefixing +: at the start of the following line." Abcjs does not seem to support this feature. I find this feature particularly useful for history fields:

```abc
X:1
T:Some tune
H:Here's the long and elaborate story of how I learnt this tune and this line
+:would indeed be very long if I couldn't use the field continuation feature 
+:of the ABC standard.
C:G
ABc def|]
```

I tried this in the [abcjs demo editor](https://www.abcjs.net/abcjs-editor) by adding an H: field to the default Cooley notation:

```abc
X: 1
T: Cooley's
M: 4/4
L: 1/8
R: reel
H:Here's some info about
+:this tune.
K: Emin
|:D2|EB{c}BA B2 EB|~B2 AB dBAG|FDAD BDAD|FDAD dAFD|
EBBA B2 EB|B2 AB defg|afe^c dBAF|DEFD E2:|
|:gf|eB B2 efge|eB B2 gedB|A2 FA DAFA|A2 FA defg|
eB B2 eBgB|eB B2 defg|afe^c dBAF|DEFD E2:|
```

This results in the following output:

<img width="771" alt="image" src="https://github.com/paulrosen/abcjs/assets/145744/4c8cefd4-2971-4ca1-977c-ff045b99f08f">

**Comment by paulrosen** [comment]
That's true - it's not implemented yet. A work around is to use `H:` instead of the plus:
```
H:Here's the long and elaborate story of how I learnt this tune and this line
H:would indeed be very long if I couldn't use the field continuation feature 
H:of the ABC standard.
```

**Comment by seisiuneer** [comment]
I've implemented a somewhat inelegant but effective fix for this in my own version of abcjs used for my ABC Transcription Tools:

https://michaeleskin.com/abctools/abctools.html

Here's how your version of Cooley's now renders in my tool:

X: 1
T: Cooley's
M: 4/4
L: 1/8
R: reel
H:Here's some info about
+:this tune.
K: Emin
|:D2|EB{c}BA B2 EB|~B2 AB dBAG|FDAD BDAD|FDAD dAFD|
EBBA B2 EB|B2 AB defg|afe^c dBAF|DEFD E2:|
|:gf|eB B2 efge|eB B2 gedB|A2 FA DAFA|A2 FA defg|
eB B2 eBgB|eB B2 defg|afe^c dBAF|DEFD E2:|

<img width="946" height="674" alt="Image" src="https://github.com/user-attachments/assets/964032c1-36d1-418b-9e2d-3e0255870efa" />
